docs/lua/functions-4.md

```
@@ -1231,18 +1231,19 @@ Checks whether Mario can become bubbled under certain game conditions (multiplay
 Transitions Mario into a bubbled state (if available in multiplayer), decrementing lives and preventing normal movement
 
 ### Lua Example
-`mario_set_bubbled(m)`
+`mario_set_bubbled(m, stayAlive)`
 
 ### Parameters
 | Field | Type |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)